plugins/adr-common/api-report.md

```diff
@@ -0,0 +1,35 @@
+## API Report File for "@backstage/plugin-adr-common"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { Entity } from '@backstage/catalog-model';
+import { IndexableDocument } from '@backstage/plugin-search-common';
+
+// @public
+export interface AdrDocument extends IndexableDocument {
+  date?: string;
+  status?: string;
+}
+
+// @public
+export type AdrFilePathFilterFn = (path: string) => boolean;
+
+// @public
+export const ANNOTATION_ADR_LOCATION_DIR = 'adr/location-dir';
```

Comment:
Do you think this would make sense without the `_DIR`/`-dir` suffix? Or perhaps some other combination.

Thinking some other options, like `adr/location`, `adr/dir`, `architecture-decisions/location`, `adr/url`